modules/bpi/bpi.module

```diff
@@ -318,14 +319,17 @@ function bpi_form_node_form_alter(&$form, &$form_state, $form_id) {
 
     // Guessing where to put 'body' text in the form.
     $bpi_body = isset($bpi_content['body']) ? $bpi_content['body'] : '';
-    if (isset($form[$body][$current_language][0]['value'])) {
-      $form[$body][$current_language][0]['value']['#default_value'] = $bpi_body;
-    }
-    elseif (isset($form[$body][$current_language][0])) {
-      $form[$body][$current_language][0]['#default_value'] = $bpi_body;
-    }
 
-    if ($bpi_id) {
+    $syndicated_images = bpi_get_syndicated_images();
+    if (is_array($syndicated_images)) {
```

Comment:
The handling of `bpi_get_syndicated_images()` which can either return an array, NULL or FALSE seems fishy to be. Is there anything we can do to improve this?